revolv/static/js/base/project.js

```diff
@@ -0,0 +1,97 @@
+$(document).ready(function () {
+	draw()
+});
+
+$( window ).resize(function() {
+    $(".project-badge-circle-grouping").remove();
+    $(".project-badge-partial-grouping").remove();
+    $(".project-badge-circle").remove();
+    $(".project-badge-line" ).remove();
+    $(".outside-circle" ).remove();
+    draw();
+});
+
+var setTextSize = function(radius) {
+    
+    var percentage_size = 3*radius/100;
+    var funded_size = percentage_size*0.5;
+
+    // d3.select(".percentage-text").attr("font-size", percentage_size + "rem")
+    // d3.select(".funded").attr("font-size", funded_size + "rem")
+
+    d3.select(".percentage-text").attr("style", "font-size:" + percentage_size + "rem")
+    d3.select(".funded").attr("font-size", "font-size:" + funded_size + "rem")
+
+
+    // percentage_text.css("font-size", percentage_size + "rem")
+    // funded_size.css("font-size", funded_size + "rem")
+}
+
+var resizeIframe = function() {
+    d3.select("iframe").attr("height", $(document).width()*0.4 + "")
+}
+
+var draw = function() {
+    // var radius = 100,
+    //     padding = 10;
+
+
+    // if ($(document).width() > 200 && $(document).width() < 800) {
+    //     radius = 48;
+    // }
+    var radius = $(document).width()*0.12;
+    setTextSize(radius)
+
+    resizeIframe();
+
+    // <circle class="outside-circle" cx="50%" cy="50%" r="" fill="white" />
```

Comment:
This comment is okay for a reference I think, but the other comments in this function can be deleted